services/catalog/src/simcore_service_catalog/api/rest/_services.py

```diff
@@ -105,56 +108,67 @@ async def list_services(
         )
 
     # now get the executable or writable services
-    services_in_db = {
-        (s.key, s.version): s
-        for s in await services_repo.list_services(
-            gids=[group.gid for group in user_groups],
-            execute_access=True,
-            write_access=True,
-            combine_access_with_and=False,
-            product_name=x_simcore_products_name,
-        )
-    }
+    with _tracer.start_as_current_span("services_repo.list_services") as span:
+        span.set_attribute("osparc-gids", f"{[group.gid for group in user_groups]}")
+        span.set_attribute("osparc-productname", f"{x_simcore_products_name}")
```

Comment:
THOUGHT: This trace is a good usage example, but I was wondering if it serves a specific purpose. For instance, is it being used for debugging something in particular? My concern is that, on one hand, in this PR  specs are being added to **remove** traces, while on the other, new traces are being introduced that might not be utilized by anyone.

Reply:
It is true. For now I will try to merge this to have at least one example in the code, but as @sanderegg also highlighted we'd need some sort of strategy for tracing if we use it more heavily, for example wrt naming conventions, what to keep, and so on.